### PR TITLE
fix: allow only sweeping calendar or project notes task in templates

### PR DESCRIPTION
**Pull request by weyert**
Introduces the option `noteTypes` which accepts `calendar` or `note` to allow cherry picking the kind of notes should be used for sweeping tasks in templates

### CODE REVIEW

**Comment by weyert** [comment]
cc @nmn

**Review by dwertheimer — CHANGES_REQUESTED** [review]
I like the addition. Pls review the Type comment and test /sw7 and /swa. The plugin is currently not working.

File: nmn.sweep/src/sweepAll.js [hunk 1]
```diff
@@ -44,10 +46,18 @@ export async function sweepTemplate(paramStr: string): Promise<string> {
       'includeHeadings',
       false,
     )
+    const noteTypes: boolean = await getTagParamsFromString(
```
[the remark this answers is not in the document]
Comment (weyert) [reply nmn.sweep/src/sweepAll.js:49]: Whoops

[the remark this answers is not in the document]
Comment (weyert) [reply nmn.sweep/src/sweepAll.js]: Changed it to `NoteTypes[]`

**Pull request merged**